pkg/apis/dashboard/migration/migrate_test.go

```diff
@@ -0,0 +1,78 @@
+package migration_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/grafana/pkg/apis/dashboard/migration"
+	"github.com/grafana/grafana/pkg/apis/dashboard/migration/schemaversion"
+)
+
+const INPUT_DIR = "testdata/input"
+const OUTPUT_DIR = "testdata/output"
+
+// Use the UPDATE_SNAPSHOTS=true environment variable to update the output files.
```

Comment:
🤔  we may not want to make that too easy... since i think we want to make sure the *output* comes from the existing typescript execution

Reply:
👍 i'll remove